app/src/main/kotlin/com/twain/interprep/data/ui/ProfileSettingsData.kt

```diff
@@ -11,15 +11,15 @@ object ProfileSettingsData {
         val imageVector: ImageVector? = null,
         @DrawableRes val imageRes: Int? = null,
         @StringRes val title: Int,
-        val label: String? = null,
+        val label: String,
```

Comment:
Also in ProfileViewModel and ProfileScreen, the associated nullable values should be non-null then